docs/examples/demo-select2-with-bootstrap.html

```diff
@@ -40,7 +40,7 @@
 
           <ui-select ng-model="ctrl.person.selected" theme="select2" class="form-control" title="Choose a person">
             <ui-select-match placeholder="Select or search a person in the list...">{{$select.selected.name}}</ui-select-match>
-            <ui-select-choices group-by="'group'" repeat="item in ctrl.people | filter: $select.search">
+            <ui-select-choices group-by="'age'" repeat="item in ctrl.people | filter: $select.search">
```

Comment:
@user378230 - No, there is no `group` property so I changed it to `age` to fix that.  There is a `country` property on these I believe.  Would it be better to use that?  `people` seem to have `name`, `email`, `age`, and `country`.